apistructs/extention.go

```diff
@@ -228,7 +228,8 @@ type ExtensionVersionQueryRequest struct {
 	Name       string
 	YamlFormat bool `query:"yamlFormat"`
 	//默认false查询有效版本, true查询所有版本
-	All string `query:"all"`
+	All                string `query:"all"`
+	OrderByVersionDesc bool   `query:"orderByVersionDesc"`
```

Comment:
bad arg.
dicehub can judge which version is default by itself, no need this arg.